libs/ledger-live-common/src/__tests__/api/socket.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  openTransportReplayer,
+  RecordStore,
+} from "@ledgerhq/hw-transport-mocker";
+import { toArray } from "rxjs/operators";
+import scenarios from "./test-helpers/scenarios";
```

Comment:
😈 import after the lib import bellow

Reply:
Shouldn't this be caught by ... something? 😪

Reply:
😢